### ISSUE TITLE
unblessed_bool doesn't DWIM (or match the docs).

### ISSUE THREAD

**Issue by dnfm** [rurban/Cpanel-JSON-XS#219]
The docs say: 

> If $enable is true (or missing), then decode will return Perl non-object boolean variables (1 and 0) for JSON booleans

However unblessed_bool causes the module to return an empty string in place of 'false':

```
$ perl -MData::Dumper -MCpanel::JSON::XS -wle 'print Dumper(Cpanel::JSON::XS->new->unblessed_bool->decode(qq/{"alpha":false,"beta":true}/));'
$VAR1 = {
          'alpha' => '',
          'beta' => 1
        };

```

**Comment by rurban** [comment]
So maybe the docs should be fixed here

**Comment by rurban** [comment]
Fixed in 4.38
